.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Proposed Commit Message
+<!-- Include a proposed commit message because all PRs can be merged in a variety of ways by the reviewer -->
+
+> summary: no more than 70 characters
+>
+> A description of what the change being made is and why it is being
```

Comment:
Requirements should be a bullet list. 

- what is the change?
- why is it made?
- a blank line is required
- wrap at 72 chars, if your text is wider than this sentence then please wrap moar
- use multiple paragraphs as appropriate